ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.txt

```diff
@@ -21,111 +21,120 @@ hideall "--sync--"
-306.0 "Wings Of Salvation x3" Ability { id: "26CA", source: "Fang of Light" } duration 8
-323.3 "Ravensbeak" Ability { id: "26B6", source: "Nael deus Darnus" }
-
-328.5 "Marker 1"
-332.5 "Marker 2"
-333.5 "Hypernova x4" duration 6 #Ability { id: "26BF", source: "Nael deus Darnus" }
-336.5 "Marker 3"
-339.5 "Cauterize" #Ability { id: "26CD", source: "Thunderwing" }
+330.2 "Fireball (4)" Ability { id: "26C5", source: "Firehorn" }
+331.2 "Doom x3" Ability { id: "26C9", source: "Tail of Darkness" }
+333.2 "Chain Lightning x2" Ability { id: "26C8", source: "Thunderwing" }
+333.3 "Wings Of Salvation x3" Ability { id: "26CA", source: "Fang of Light" } duration 8
+350.6 "Ravensbeak" Ability { id: "26B6", source: "Nael deus Darnus" }
+
+355.8 "Marker 1"
+359.8 "Marker 2"
+360.8 "Hypernova x4" duration 6 #Ability { id: "26BF", source: "Nael deus Darnus" }
+363.8 "Marker 3"
+366.8 "Cauterize" #Ability { id: "26CD", source: "Thunderwing" }
 #339.5 "Cauterize" #Ability { id: "26CE", source: "Tail of Darkness" }
-341.0 "--untargetable--"
-341.0 "Meteor/Dive or Dive/Beam" duration 3 # first mechanic -> second
+368.3 "--untargetable--"
+368.3 "Meteor/Dive or Dive/Beam" duration 3 # first mechanic -> second
 #342.7 "Meteor Stream" Ability { id: "26C0", source: "Nael Geminus" }
-343.5 "Cauterize" #Ability { id: "26CB", source: "Firehorn" }
+370.8 "Cauterize" #Ability { id: "26CB", source: "Firehorn" }
 #345.7 "Dalamud Dive" Ability { id: "26C1", source: "Nael deus Darnus" }
-347.4 "Cauterize" #Ability { id: "26CC", source: "Iceclaw" }
+374.7 "Cauterize" #Ability { id: "26CC", source: "Iceclaw" }
 #347.5 "Cauterize" #Ability { id: "26CF", source: "Fang of Light" }
 
-349.7 "Bahamut's Claw x5" duration 2.8 #Ability { id: "26B5", source: "Nael deus Darnus" }
+377.0 "Bahamut's Claw x5" duration 2.8 #Ability { id: "26B5", source: "Nael deus Darnus" }
 
-361.2 "Random Combo Attack" duration 8
+388.5 "Random Combo Attack" duration 8
 #366.2 "Thermionic Beam" Ability { id: "26BD", source: "Nael deus Darnus" }
 #369.2 "Lunar Dynamo" Ability { id: "26BC", source: "Nael deus Darnus" }
 
-372.7 "Random Combo Attack" duration 8
+400.0 "Random Combo Attack" duration 8
 #377.3 "Raven Dive" Ability { id: "26BE", source: "Nael deus Darnus" }
 #380.3 "Lunar Dynamo" Ability { id: "26BC", source: "Nael deus Darnus" }
 
-388.7 "Ravensbeak" Ability { id: "26B6", source: "Nael deus Darnus" }
-395.7 "Bahamut's Claw x5" duration 2.8 #Ability { id: "26B5", source: "Nael deus Darnus" }
-403.2 "--untargetable--"
-408.2 "Megaflare Enrage" Ability { id: "26BA", source: "Nael deus Darnus" }
+416.0 "Ravensbeak" Ability { id: "26B6", source: "Nael deus Darnus" }
+423.0 "Bahamut's Claw x5" duration 2.8 #Ability { id: "26B5", source: "Nael deus Darnus" }
+430.5 "--untargetable--"
+435.5 "Megaflare Enrage" Ability { id: "26BA", source: "Nael deus Darnus" }
 
+# Detect Nael going untargetable for phase change
+494.7 "--sync--" NameToggle { name: "Nael deus Darnus", toggle: '00' } window 250,0
```

Comment:
Would this potentially catch Nael going untargetable during dive bombs (e.g. at `368.3`)?  (IIRC, you can skip dive bombs too, so if `NameToggle` would catch either, that might be problematic.)

Reply:
`NameToggle` doesn't get sent for divebombs. Not sure why, maybe just ARR jank, but it's not an issue here at least.

Here's every `NameToggle` `Nael deus Darnus` line for a pull that saw divebombs and Bahamut Prime phase transition, along with the `Cauterize` divebomb casts for timing purposes.

```
34|2024-06-06T22:38:59.3970000-04:00|400137E2|Nael deus Darnus|400137E2|Nael deus Darnus|01|e517c92963846eff
20|2024-06-06T22:41:04.1990000-04:00|40013A8E|Thunderwing|26CD|Cauterize|40013A8E|Thunderwing|3.700|-0.02|-24.00|0.00|0.13|b3279d922298c2ea
20|2024-06-06T22:41:04.2410000-04:00|40013A8D|Tail of Darkness|26CE|Cauterize|40013A8D|Tail of Darkness|3.700|16.65|-16.68|0.00|-0.51|f50f6b80e09b7844
20|2024-06-06T22:41:08.1180000-04:00|40013A8F|Iceclaw|26CC|Cauterize|40013A8F|Iceclaw|3.700|16.65|16.65|0.00|-1.93|2233a0375f760ad7
20|2024-06-06T22:41:12.0830000-04:00|40013A8C|Fang of Light|26CF|Cauterize|40013A8C|Fang of Light|3.700|-16.68|-16.68|0.00|0.11|ec0f8a7ca8b76f1a
20|2024-06-06T22:41:12.1720000-04:00|40013A90|Firehorn|26CB|Cauterize|40013A90|Firehorn|3.700|-24.00|-0.02|0.00|1.26|1285e1e52798f4c6
34|2024-06-06T22:41:41.9850000-04:00|400137E2|Nael deus Darnus|400137E2|Nael deus Darnus|00|6779ffb61295abf0
22|2024-06-06T22:41:47.2840000-04:00|40013811|Bahamut Prime|26D1|Seventh Umbral Era|<trim for privacy>
```